core/templates/dev/head/pages/profile/Profile.js

```diff
@@ -126,14 +126,24 @@ oppia.controller('Profile', [
           $log.error('Error: cannot decrement page');
         } else {
           $scope.currentPageNumber--;
+          $scope.startingExplorationNumber = ($scope.currentPageNumber *
+                                              $scope.PAGE_SIZE + 1);
+          $scope.endingExplorationNumber = (
+            Math.min($scope.numUserPortfolioExplorations,
+              ($scope.currentPageNumber + 1) * $scope.PAGE_SIZE));
         }
       };
       $scope.goToNextPage = function() {
-        if ($scope.currentPageNumber * $scope.PAGE_SIZE >= (
+        if (($scope.currentPageNumber + 1) * $scope.PAGE_SIZE >= (
           data.edited_exp_summary_dicts.length)) {
           $log.error('Error: Cannot increment page');
         } else {
           $scope.currentPageNumber++;
+          $scope.startingExplorationNumber = ($scope.currentPageNumber *
+                                              $scope.PAGE_SIZE + 1);
```

Comment:
Break at the first parenthesis here and elsewhere
```javascript
$scope.startingExplorationNumber = (
     ...
);
```

Reply:
done..!